Model/src/CardPackage/Deck.java

```diff
@@ -0,0 +1,4 @@
+package CardPackage;
+
+public class Deck {
+}
```

Comment:
Same thing as Card Class, do we want a Deck class or a ChanceSquare that builds and holds an array of ChanceCard objects?

Reply:
This is just a class that is used to construct a deck of cards. In the main program, there would be two instances of Deck. 

Example

private Deck ChanceCardDeck;

private Deck CommunityCheckCardDeck;